RELEASE.adoc

```diff
@@ -91,6 +91,12 @@ Another example is if a security audit is done on `rc1`, and the changes in `rc2
 - Update the version bounds in cabal files.
 - Update the CHaP index state in `cabal.project`.
 - Update the CHaP flake input with `nix flake update CHaP`.  If you get "error: cannot find flake 'flake:CHaP' in the flake registries" your nix installation probably needs to be updated.
+- Update the package versions of affected packages in the cabal files, if needed.
+  Ledger packages follow a different versioning policy than Plutus packages, where each package is versioned independently.
+  Each ledger PR is responsible for incrementing the version of any affected ledger package to the next release version (unless this has already been done by a previous PR).
+  For a PR bumping Plutus version bounds, if there are no other breaking changes, increment the last component of the ledger package versions.
```

Comment:
What does "last component" mean? Does it refer to a dependency hierarchy?

Reply:
Oh I think I understand, it's referring to the structure of the version identifier, like the last component of "1.2.3.4" would be "4". Is that right?

I got confused by the discussion here: https://github.com/IntersectMBO/cardano-ledger/pull/4593/files#r1744246536 . Also, from this and from reading `cardano-ledger`'s release guide I understand that all packages should be updated? Waiting on a  response on that PR to be sure.